### PR TITLE
Add script to check SDL does not resend OnSeekMediaClockTimer if wron…

### PR DESCRIPTION
**Pull request by OVikhrov**
…g json syntax
@aarotar please review

### CODE REVIEW

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_Wrong_Json_Syntax.lua [hunk 1]
```diff
@@ -0,0 +1,94 @@
+--------------------------------------------------------------------------------------------
+-- Requirement summary:
```
Comment (aarotar) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_Wrong_Json_Syntax.lua]: @OVikhrov, add to requirement summary also CRQ related to OnSeekMediaClockTimer

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_Wrong_Json_Syntax.lua [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+commonSteps:ActivateAppInSpecificLevel(self, self.applications["Test Application"])
+EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL", systemContext = "MAIN"})
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+function Test:TestStep_OnSeekMediaClockTimer_With_Wrong_Json_Syntax()
+  -- missing ':' after "seconds"
+  self.hmiConnection:Send('{"jsonrpc":"2.0","method":"UI.OnSeekMediaClockTimer","params":{"seekTime":{"seconds" 30,"minutes":30,"hours":30},"appID":' .. self.applications["Test Application"] ..'}}')
+
```
Comment (aarotar) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_Wrong_Json_Syntax.lua]: @OVikhrov, plase avoid redundant empty lines

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_Wrong_Json_Syntax.lua (hunk 1 above)
Comment (OVikhrov) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_Wrong_Json_Syntax.lua]: Already, string below

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_Wrong_Json_Syntax.lua (hunk 2 above)
Comment (OVikhrov) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_Wrong_Json_Syntax.lua]: Accepted

**Comment by OVikhrov** [comment]
@aarotar Plese review iteration#2

**Pull request merged**